include/openssl/ssl.h

```diff
@@ -1361,7 +1361,8 @@ struct ssl_private_key_method_st {
 };
 
 // SSL_set_private_key_method configures a custom private key on |ssl|.
-// |key_method| must remain valid for the lifetime of |ssl|.
+// |key_method| must remain valid for the lifetime of |ssl|. Using custom
+// keys with the multiple certificate slots feature is not supported.
```

Comment:
This comment was also backported from the original commit: https://github.com/google/boringssl/commit/d1d8078025cc654d60ee873c88740f8a987a77b7
It's a BoringSSL specific function that doesn't exist in OpenSSL, so adding multiple cert slots support for this function shouldn't be a priority for now.